src/components/core/core.js

```diff
@@ -322,9 +322,9 @@ export default class Core extends UIObject {
     this.getCurrentContainer().trigger(showing?Events.CONTAINER_MEDIACONTROL_SHOW:Events.CONTAINER_MEDIACONTROL_HIDE)
 
     if (showing)
-      this.$el.removeClass('nocursor')
+    {this.$el.removeClass('nocursor')}
```

Comment:
One of the things we want to avoid is doing this. We should either have the braces in "1tbs" style or have a single-line if statement, preferably with no braces.

Reply:
:+1: